src/check-branches-match.js

```diff
@@ -0,0 +1,19 @@
+/*
+  checkBranch
+  ====
+  - check that the arguments are defined
+  - check if the git branch matches the ci branch
+*/
+
+module.exports = (gitBranch, ciBranch, errorCallback) => {
```

Comment:
only calling a callback on error is confusing. just return a promise

Reply:
Something like this? I don't know what the code is supposed to do

```js
module.exports = (gitBranch, ciBranch, errorCallback) => new Promise((resolve, reject) => {
  if (typeof gitBranch === 'undefined') {
    console.error(`Harmless Changes: ${gitBranch} is undefined.`)
    return reject()
  } else if (typeof ciBranch === 'undefined') {
    console.error(`Harmless Changes: ${ciBranch} is undefined.`)
    return reject()
  } else if (gitBranch === ciBranch) {
    console.error(`Harmless Changes: The git branch ${gitBranch} matches the ci branch ${ciBranch}. Better Test!`)
    return reject()
  } 
  console.warn(`Harmless Changes: The git branch ${gitBranch} does not match the ci branch ${ciBranch}. Moving on!`)
  return resolve()
})
```

Reply:
Jest fails with `process.exit(1)`. I don't know why and I have looked for an even longer time now after the comment above. ...That is why I used the error callback. It is stub for `process.exit(1)`.